features/share/smart-contracts/src/GqlMutations.js

```diff
@@ -26,13 +27,14 @@ export class GqlMutations {
               input: {
                 portfolioIdentifier: "AAAA1111",
                 portfolioName: "Loan Portfolio"
-            }
+            }.
```

Comment:
This period should be a comma

Reply:
Updated (luckily our GQL parser ignores punctuation between lines)

Reply:
Interesting, because when I ran the data loader it failed to create the portfolio and when I added a log statement to output the error it showed it was due to the period.  Actually the log statement was helpful, I will share it in an additional comment.